server/config.go

```diff
@@ -107,8 +123,9 @@ type Config struct {
 // NewConfig returns an instance of Config with default options.
 func NewConfig() *Config {
 	c := &Config{
-		DataDir:             "~/.pilosa",
-		Bind:                ":10101",
+		DataDir:    "~/.pilosa",
+		ListenAddr: ":10101",
+		// AdvertiseAddr: "",
```

Comment:
well, I think because this was a place where defaults are being defined, we were originally listing all attributes, along with the default value, and commenting out those lines which use the go default value. but it doesn't look like this pattern has been maintained, and this is the second time someone has asked about this, so i'll just remove them.

Reply:
I removed all these commented lines.